indigo_app/views/tasks.py

```diff
@@ -398,6 +411,38 @@ def get_redirect_url(self):
         return reverse('task_detail', kwargs={'place': self.kwargs['place'], 'pk': self.kwargs['pk']})
 
 
+class TaskChangeBlockingTasksView(SingleTaskViewBase, View, SingleObjectMixin):
+    # permissions
+    permission_required = ('indigo_api.change_task',)
+
+    http_method_names = ['post']
+
+    def post(self, request, *args, **kwargs):
+        task = self.get_object()
+        user = self.request.user
+        task.updated_by_user = user
+        ids = self.request.POST.getlist('blocked_by')
+
+        if ids:
+            blocked_by = Task.objects.filter(id__in=ids).all()
```

Comment:
Also filter by the current country and locality.

Reply:
Is this a performance thing? because tasks have unique ids across places, right? Happy to do it of course, just curious

Reply:
It's about being defensive about what you accept and validating the inputs. The user should only be able to specify task ids of tasks in the same place as the current task.